obstacle_range: 3.0
raytrace_range: 3.5

footprint: [[-0.105, -0.105], [-0.105, 0.105], [0.041, 0.105], [0.041, -0.105]]
#robot_radius: 0.105

obstacle_layer:
  observation_sources: LaserScan
  LaserScan: {sensor_frame: base_laser_link, data_type: LaserScan, topic: scan, marking: true, clearing: true}
  max_obstacle_height: 0.6
  min_obstacle_height: 0.0
  obstacle_range: 4.0
  raytrace_range: 4.0

inflation_layer:
  inflation_radius: 0.4
  cost_scaling_factor: 10